src/com/exolius/simplebackup/Commands.java

```diff
@@ -16,17 +17,17 @@ public Commands(SimpleBackup plugin) {
     --------------------------------------------------------*/
     @Override
     public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
-        if (cmd.getName().equalsIgnoreCase("sbackup")) {
-            if (sender.hasPermission("simplebackup.use")) {
-                new Thread(new Runnable() {
-                    @Override
-                    public void run() {
-                        plugin.doBackup();
-                    }
-                }).start();
-            }
-            return true;
+        if (sender.hasPermission("simplebackup.use")) {
+            new Thread(new Runnable() {
+                @Override
+                public void run() {
+                    plugin.doBackup();
+                }
+            }).start();
+        } else {
+        	sender.sendMessage(ChatColor.RED + "You don't have permission to execute this command.");
         }
-        return false;
```

Comment:
You're aware that returning false prints the usage message in plugin.yml, and if the player doesn't have permission, that can also be specified in the plugin.yml. Albeit he doesn't use it for the latter purpose, I don't really see the need to tell a player they don't have permission for a simple command like this (which the player shouldn't know about anyways...)

Feels more like a customization thing anyways.